operators/pkg/utils/test/integration.go

```diff
@@ -45,10 +46,22 @@ func RunWithK8s(m *testing.M, crdPath string) {
 		ControlPlaneStartTimeout: ControlPlaneStartTimeout,
 	}
 
-	var err error
-	if Config, err = t.Start(); err != nil {
-		fmt.Println("failed to start test environment:", err.Error())
-		panic(err)
+	// attempt to start the k8s test environment
+	// this happens to fail for various reasons, let's retry up to BootstrapTestEnvRetries times
+	retry := 1
+	for {
+		if retry > BootstrapTestEnvRetries {
+			fmt.Printf("failed to start test environment after %d attempts, exiting.\n", BootstrapTestEnvRetries)
```

Comment:
If the retries are not enough we lose the reason why the start failed. It could be useful to log the last error?

Reply:
I think the error is already logged for each individual attempt? (see a few lines below)